srcpkgs/python3-orocos-kdl/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-orocos-kdl'
+pkgname=python3-orocos-kdl
```

Comment:
needs to define PYTHON_VERSION -DPYTHON_VERSION=3

Reply:
I'm assuming you mean in configure_args.